app/scripts/behaviors/datetime-behavior.js

```diff
@@ -6,5 +6,42 @@ Behaviors.DateTimeBehavior = {
 			return '';
 		}
 		return util.formatDate(date);
-	}
+	},
+	_getDate: function(d){
+		var date = new Date(d * 1000);
+		var result = date.toISOString();
+		return result.substring(0, result.indexOf('T'));
+	},
+	_getTime: function(d){
+		var date = new Date(d * 1000);
+		var hours = date.getUTCHours();
+		var mins = date.getUTCMinutes();
+		var time = '';
+		if(hours < 10){
+			time = time + '0';
+		}
+		time = time + hours + ':';
+		if(mins < 10){
+			time = time + '0';
+		}
+		time = time + mins;
+		return time;
+	},
+	_computeDuration: function(duration) {
```

Comment:
we should start migrating to behaviors. Now i added it here and in another pr i`ll replace the util usages